### ISSUE TITLE
Too many Government Plaza buildings listed

### ISSUE THREAD

**Issue by kblease** [kblease/ProductionQueue#47]
Government Plaza buildings are currently not checking for the requirement of having a sufficiently advanced government type. It doesn't appear that this is made abundantly clear it is a requirement in the Civilopedia, but it is indeed the case.

**Comment by Question2** [comment]
Because of this bug, i cannot build -any- government plaza buildings at all...the UI just glitches out when i try and the buildings are not queued.

**Comment by kblease** [comment]
In order to build Government Plaza buildings, you must have a certain government level. At the moment the mod does not check for this and shows them as queueable in the production panel. Once a queued building that cannot actually be built arrives at the top of the queue, things stop working properly until you start production on a valid item. _If_ you were to have the appropriate government level, the Government Plaza buildings would be able to be built. 

Basically this is a minor bug that's causing way too much confusion so I'm about to fix it.
